src/pykeen/nn/combinations.py

```diff
@@ -178,26 +174,42 @@ def __init__(
         :param entity_embedding_dim: The dimension of the entity representations to which literals are concatenated
         :param literal_embedding_dim: The dimension of the literals that are concatenated
         :param input_dropout: The dropout probability of an element to be zeroed.
-        :param activation: The activation function, resolved by :data:`pykeen.utils.activation_resolver`.
+        :param activation:
+            The activation function, resolved by :data:`class_resolver.contrib.torch.activation_resolver`.
 
         This class uses a :class:`torch.nn.Tanh` by default for the activation to the :class:`LinearDropout` as
         described by [kristiadi2018]_.
         """
-        super().__init__(
-            real_module=LinearDropout(
-                entity_embedding_dim=entity_embedding_dim,
-                literal_embedding_dim=literal_embedding_dim,
-                input_dropout=input_dropout,
-                activation=activation,
-            ),
-            imag_module=LinearDropout(
-                entity_embedding_dim=entity_embedding_dim,
-                literal_embedding_dim=literal_embedding_dim,
-                input_dropout=input_dropout,
-                activation=activation,
-            ),
+        super().__init__()
+        self.real_mod = LinearDropout(
+            entity_embedding_dim=entity_embedding_dim,
+            literal_embedding_dim=literal_embedding_dim,
+            input_dropout=input_dropout,
+            activation=activation,
+        )
+        self.imag_mod = LinearDropout(
+            entity_embedding_dim=entity_embedding_dim,
+            literal_embedding_dim=literal_embedding_dim,
+            input_dropout=input_dropout,
+            activation=activation,
         )
 
+    def forward(self, xs: Sequence[torch.FloatTensor]) -> torch.FloatTensor:
+        """Split the complex vector, combine the representation parts and literal, score, then recombine."""
+        assert len(xs) == 2
+        x, literal = xs
+        x_re, x_im = split_complex(x)
+        x_re = self.real_mod(torch.cat([x_re, literal], dim=-1))
+        x_im = self.imag_mod(torch.cat([x_im, literal], dim=-1))
+        return combine_complex(x_re=x_re, x_im=x_im)
+
+    # docstr-coverage: inherited
+    def output_shape(self, input_shapes: Sequence[Tuple[int, ...]]) -> Tuple[int, ...]:  # noqa: D102
+        # symbolic output to avoid dtype issue
+        # we only need to consider real part here
+        # FIXME should score_real be real_mod?
```

Comment:
FIXME.... looks like these aren't being covered by tests

Reply:
fixed and covered by tests